Makefile

```diff
@@ -6,7 +6,7 @@ VERSION ?= $(shell git describe --match 'v[0-9]*' --dirty='.m' --always)
 REVISION ?= $(shell git rev-parse HEAD)$(shell if ! git diff --no-ext-diff --quiet --exit-code; then echo .m; fi)
 
 
-PKG=github.com/docker/distribution
+PKG=github.com/docker/distribution/v3
```

Comment:
Missed this one, which caused some paths not being excluded, what caused CI to fail.

all green now